YouVsVirus/Assets/Scenes/leveldemo/EndLevelControllerLeveldemo.cs

```diff
@@ -19,10 +19,12 @@ protected override void Update()
     {
         base.Update();
         // additonally we can fail this level if we are infected by propaganda smileys
-        if (playerInfectedByPropaganda && !playerExposed)
+        if (!levelHasFinished && playerInfectedByPropaganda && !playerExposed)
         {
             CreateHumans.GetComponent<CreatePopLeveldemo>().CummulativeSpriteUpdate();
             CanvasProp.SetActive(true);
+            FindAndPlaceHuman(CanvasProp);
+            levelHasFinished = true;
         }
         // make sure that not both canvases appear
```

Comment:
Why not use else if? This would ensure that only one of the cases can occur.

Reply:
It was no longer needed, so this is done